README.md

```diff
@@ -30,3 +31,27 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+T(n)=1 if n<=1
+3T(n/3) + n^5 else
+
+Solve by substitution 
+
+$T(n) = 3T(\frac{n}{3}) + n^5$<br>
+$T(n) = 3(3T(\frac{n/3}{3}) + (\frac{n}{3})^5) + n^5$<br>
+$T(n) = 3^2T(\frac{n}{3^2}) + \frac{n^5}{3^4} + n^5$<br>
+$T(n) = 3^3T(\frac{n}{3^3}) + \frac{n^5}{3^5} + \frac{n^5}{3^4} + n^5$<br>
+
+Looking at the pattern, we can see the recurrence relation is
+
+$3^iT(\frac{n}{3^i}) + \displaystyle\sum_{j=0}^i \frac{n^5}{3^{i+2}}  + n^5$.<br>
```

Comment:
The sum you give here does not allow you to get the terms you have above. It cannot be correct.